build.sbt

```diff
@@ -2717,11 +2717,12 @@ val allStdBits: Parser[String] =
 lazy val `simple-httpbin` = project
   .in(file("tools") / "simple-httpbin")
   .settings(
-    frgaalJavaCompilerSetting,
```

Comment:
The frgaal compiler settings have been causing compile error:
`cannot find symbol import com.sun.net.httpserver.SimpleFileServer: cannot find symbol`.

Apparently `SimpleFileServer` may not be in the JDK that Frgaal uses? Weird.

Anyway there is no benefit in Frgaal within this sub-project, especially as we are already on JDK21. Removing it allowed me to re-use the `SimpleFileServer` for serving files for testing, instead of having to roll my own solution or having to bring in some heavy dependencies.

Reply:
> > server is running fine on our JDK.
> 
> You have compiled your code in a way that allows running it on JDK 11. However your code cannot run on JDK 11. Without Frgaal you wouldn't even notice that your project is misconfigured as it requires JDK18+ to run.

Hm, I see. It was hard to appreciate that, because from my perspective, removing Frgaal made it work - I never really specifically wanted to compile `simple-httpbin` to run on JDK 11 - just the current JDK we use - especially since it's not even an artifact we ship but just an internal test scaffolding.